woltka/cli.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020--, Qiyun Zhu.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""Standalone command-line interface (CLI) of the program.
+"""
+
+import click
+
+from . import __version__
+from .workflow import workflow
+
+
+class NatOrder(click.Group):
+    """Natural ordering of click command groups.
+    """
+    def list_commands(self, ctx):
+        return self.commands.keys()
+
+
+@click.version_option(__version__)
+@click.group(cls=NatOrder)
+def cli():
+    pass
+
+
+# `gotu` is a simplified wrapper of `classify`
+
+@cli.command('gotu')
+@click.option(
+    '--input', '-i', 'input_path', required=True,
+    type=click.Path(exists=True, file_okay=True, dir_okay=True),
+    help=('Path to a multiplexed alignment file or a directory of per-sample '
+          'alignment files.'))
+@click.option(
+    '--output', '-o', 'output_path', required=True,
+    type=click.Path(writable=True),
+    help=('Path to output gOTU table.'))
+@click.option(
+    '--ambig/--no-ambig', default=True,
+    help=('Allow one sequence to be assigned to multiple gOTUs. Each hit '
+          'will be counted as 1 / k (k is the totally number of hits).'))
```

Comment:
You should specify the behavior if `--no-ambig`, namely, ambiguous hits will be ignored.

Reply:
Good idea! Added one sentence.